src/SprykerEco/Service/ComputopApi/Converter/ComputopApiConverter.php

```diff
@@ -51,33 +67,42 @@ public function extractResponseHeader(array $plaintextResponseHeader, $method):
         $header->setIsSuccess($this->isStatusSuccess($header));
         $header->setMethod($method);
 
+        $macResponseEncryptedValue = $this->hmacHasher->getEncryptedValue(
+            $this->computopApiMapper->getMacResponseEncryptedValue($header),
+        );
+
+        $this->checkMacResponse(
```

Comment:
`checkMacResponse` method now used only inside of a model but still public

Reply:
exception throwing at `checkMacResponse` should be mentioned at API specs

Reply:
Fixed